### PR TITLE
{bio}[GCC/9.3.0] blasr_libcpp v5.3.4

### PR DESCRIPTION
**Pull request by mboisson**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by mboisson** [comment]
Ah, I guess this can also not be lower than the MPI level because of the HDF5 dependency ? @boegel

**Comment by mboisson** [comment]
This PR requires HDF5.Serial (https://github.com/easybuilders/easybuild-easyconfigs/pull/11155) and Boost.Serial (https://github.com/easybuilders/easybuild-easyconfigs/pull/11145)

**Comment by mboisson** [comment]
Mmm, I don't know what this failed check means : 
```AssertionError: ['sanity_pip_check is enabled in cram-0.7-GCCcore-9.3.0.eb'] is not false : sanity_pip_check is enabled in cram-0.7-GCCcore-9.3.0.eb```

**Comment by bartoldeman** [comment]
Reopen to re-run checks

File: easybuild/easyconfigs/c/cram/cram-0.7-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['7da7445af2ce15b90aad5ec4792f857cef5786d71f14377e9eb994d8b8337f2f']
+
+builddependencies = [('binutils', '2.34')]
+
+multi_deps = {'Python': ['3.8.2', '2.7.18']}
+
+download_dep_fail = True
+use_pip = True
```
Comment (akesandgren) [inline easybuild/easyconfigs/c/cram/cram-0.7-GCCcore-9.3.0.eb:20]: missing
```
sanity_pip_check = True
```
here

**Comment by akesandgren** [comment]
This one depends on HDF5.Serial and Boost.Serial which are not in devel, they where merged to dual_serial_mpi branch.

**Review by akesandgren — CHANGES_REQUESTED** [review]

**Comment by akesandgren** [comment]
Closing this PR, since it (only) uses deprecated toolchain versions,
see https://docs.easybuild.io/policies/toolchains .

**Pull request closed without merging**